userguide/jails.rst

```diff
@@ -1430,7 +1382,7 @@ Managing iocage Jails
 ~~~~~~~~~~~~~~~~~~~~~
 
 Creating a jail automatically starts the iocage configuration process
-for the %brand% system. Jail properties can also be specified with the
+for the %brand% system. Jail properties are also specified with the
```

Comment:
The original tense was correct.  Jail properties are not mandatory, just can be included.